README.md

```diff
@@ -1 +1,204 @@
+Have a laptop as well
+
+
+### Setup
+
+Clone this repository to your desired folder:
+git clone git@github.com:ismailhallirumuhammad/portfolio-mobile-first
+.git
+
+
+### Install
+
+Install this project with:
+Linters and there are many ways you can integrate a linter they are but not limited to the following:
+
+1-Text editor plugins
+2-GitHub Actions
+3-GitHub apps
+
+Instructions:
+a-Make sure that there are no linter errors:
+b-Linter should be set up correctly for your repository.
+c-Linter result should be green.
+d-There should be no custom changes to linter config.
+
+
+### Usage
+
+To run the project, execute the following command:
+To navigate through the repo you type: cd portfolio-mobile-first.,
+Open the project in an IDE
+You can also run this project using or by lunching the live server plugging in VSCode.
+
+
+### Run tests
+
+To run tests, run the following command:
+Use browser like chrome by clicking on the link or use live server built in VSCode
+
+
+### Deployment
+
+You can deploy this project using:
+github repository(Github Enterprise Cloud and Github Enterprise Server)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors">Ismail Halliru Muhammad</a>
+
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/ismailhallirumuhammad)
+- Twitter: [@twitterhandle](https://mobile.twitter.com/IsmailhalliruM1)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/mwlite/in/ismail-halliru-muhammad-2a8453127)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+-  **[Add Work and About Myself Sections.]**
+-  **[Add Links and Buttons.]**
+-  **[Add form and the like (Contact me)]**
+-  **[Add javascript to make it functional]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ismailhallirumuhammad/portfolio-mobile-first
+/issues).
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project, you can contact the owner or use it for referrence...
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank:
+The entire Microverse team for their endless effort most especially the CEO,
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **[What stack used in the project?]**
+
+  - [HTML/CSS]
+
+- **[What is the project is all about?]**
+
+  - **[Targetting devices screen size]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Good job with your README file setting 👍 , It could be better to avoid broken in this file it will make it more professional , Kindly solve the broken link attached to the MIT 😄
- [ ] [OPTIONAL] Kindly remove the note o make your project more proffesional 👍